src/pyrqlite/cursors.py

```diff
@@ -88,19 +88,96 @@ def _request(self, method, uri, body=None, headers={}):
                     indent=4))
         return response_json
 
-    def _substitute_params(self, operation, parameters):
+    def _get_sql_command(self, sql_str):
+        return sql_str.split(None, 1)[0].upper()
+    
+    def _strip_strings(self, operation):
         '''
-        SQLite natively supports only the types TEXT, INTEGER, REAL, BLOB and
-        NULL
+        This function removes string literals from the SQL operation so we
```

Comment:
@joshiggins -- this is what I don't understand then. Why modify the SQL string at all? Should it not be supplied in the SQL strings place holder in the Params API? The SQLite code will take care of it.

Reply:
Oh, I see, I had the wrong mental model for this change (I'm not super familiar with this library). This is about breaking apart the SQL entered by the client library so the API call to rqlite can be made correctly. Let me take a look now.